test-data/unit/check-redundant-expr.test

```diff
@@ -0,0 +1,17 @@
+-- Type checker test cases for conditional checks that result in some
```

Comment:
Can you move the contents of this file to `check-errorcodes.test`, since it looks like the logical place for this?